packages/frameworks-solid-start/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@auth/solid-start",
+  "description": "Authentication for SolidStart.",
+  "version": "0.0.1",
+  "type": "module",
+  "files": [
+    "handler.*",
+    "session.*",
+    "utils.*",
+    "src"
+  ],
+  "exports": {
+    "./handler": {
+      "types": "./handler.d.ts",
+      "import": "./handler.js"
+    },
+    "./session": {
+      "types": "./session.d.ts",
+      "import": "./session.js"
+    },
+    "./utils": {
+      "types": "./utils.d.ts",
+      "import": "./utils.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "tsup --config ./tsup.config.js",
+    "patch": "npm version patch --no-git-tag-version",
+    "clean": "rm -rf utils.* handler.* session.*"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@solidjs/meta": "^0.28.0",
+    "@types/node": "^18.7.14",
+    "@types/set-cookie-parser": "^2.4.2",
+    "next-auth": "^4.18.6",
+    "solid-js": "^1.5.7",
+    "solid-start": "^0.2.1",
+    "typescript": "^4.8.2",
+    "tsup": "^6.5.0"
+  },
+  "peerDependencies": {
+    "solid-js": "^1.5.7",
+    "solid-start": "^0.2.1",
+    "@auth/core": "~0.1.4 || ^0.1.4"
+  },
+  "dependencies": {
+    "set-cookie-parser": "^2.5.1"
+  },
+  "keywords": [
+    "SolidJS",
+    "SolidStart",
+    "Auth"
+  ],
+  "author": "OrJDev",
```

Comment:
You could also add your e-mail (feel free not to, if not interested)

```suggestion
  "author": "OrJDev <your@email.com>",
```

Reply:
sure, done